mysql-test/suite/innodb/t/bug84563-master.opt

```diff
@@ -0,0 +1 @@
+--log-error=$MYSQLTEST_VARDIR/tmp/bug84563.err
```

Comment:
This breaks the testcase with false positive when run with MTR --repeat option. The testcase should not use a custom error log path as an option, then restart in the testcase with the path, restart again with default path, grep & delete the custom path

Reply:
Agree. But there is not need to restart the server once more with default error log path. I will be restarted automatically for the next test case (see server_need_restart() in mysql_test_run.pl)

Reply:
Right, but you need to remove the custom error log path at the end, and the most portable way to do that is with file closed, for which server has to be restarted with a different path. But for us either way is fine.